.travis.yml

```diff
@@ -6,11 +6,7 @@ go:
   - 1.6
 
 before_install:
-  - go get github.com/golang/mock/gomock
-  - go get github.com/golang/mock/mockgen
-  - go get github.com/onsi/gomega
-  - go get github.com/onsi/ginkgo
-  - go get golang.org/x/tools/cmd/cover
+  - make dependencies
```

Comment:
so I tried go get -t $(go list./...)
and apparently that does not work correctly as input like go vet $(go list ./...) or go test $(go list ./...)
https://travis-ci.org/scootdev/scoot/builds/136274042

So we can manually list every project or ever dependency what are people's preferences